src/gui/widgets/Fader.cpp

```diff
@@ -250,102 +250,102 @@ void Fader::mouseReleaseEvent( QMouseEvent * mouseEvent )
 }
 
 
-void Fader::wheelEvent ( QWheelEvent *ev )
+void Fader::wheelEvent (QWheelEvent *ev)
 {
 	ev->accept();
 
 	if (ev->angleDelta().y() > 0)
 	{
-		model()->incValue( 1 );
+		model()->incValue(1);
 	}
 	else
 	{
-		model()->incValue( -1 );
+		model()->incValue(-1);
 	}
 	updateTextFloat();
-	s_textFloat->setVisibilityTimeOut( 1000 );
+	s_textFloat->setVisibilityTimeOut(1000);
 }
 
 
 
 ///
 /// Set peak value (0.0 .. 1.0)
 ///
-void Fader::setPeak( float fPeak, float &targetPeak, float &persistentPeak, QElapsedTimer &lastPeakTimer )
+void Fader::setPeak(float fPeak, float &targetPeak, float &persistentPeak, QElapsedTimer &lastPeakTimer)
 {
-	if( fPeak <  m_fMinPeak )
+	if(fPeak <  m_fMinPeak)
 	{
 		fPeak = m_fMinPeak;
 	}
-	else if( fPeak > m_fMaxPeak )
+	else if(fPeak > m_fMaxPeak)
 	{
 		fPeak = m_fMaxPeak;
 	}
 
-	if( targetPeak != fPeak)
+	if(targetPeak != fPeak)
 	{
 		targetPeak = fPeak;
-		if( targetPeak >= persistentPeak )
+		if(targetPeak >= persistentPeak)
 		{
 			persistentPeak = targetPeak;
 			lastPeakTimer.restart();
 		}
 		update();
 	}
 
-	if( persistentPeak > 0 && lastPeakTimer.elapsed() > 1500 )
+	if(persistentPeak > 0 && lastPeakTimer.elapsed() > 1500)
 	{
-		persistentPeak = qMax<float>( 0, persistentPeak-0.05 );
+		persistentPeak = qMax<float>(0, persistentPeak-0.05);
 		update();
 	}
 }
 
 
 
-void Fader::setPeak_L( float fPeak )
+void Fader::setPeak_L(float fPeak)
 {
-	setPeak( fPeak, m_fPeakValue_L, m_persistentPeak_L, m_lastPeakTimer_L );
+	setPeak(fPeak, m_fPeakValue_L, m_persistentPeak_L, m_lastPeakTimer_L);
 }
 
 
 
-void Fader::setPeak_R( float fPeak )
+void Fader::setPeak_R(float fPeak)
 {
-	setPeak( fPeak, m_fPeakValue_R, m_persistentPeak_R, m_lastPeakTimer_R );
+	setPeak(fPeak, m_fPeakValue_R, m_persistentPeak_R, m_lastPeakTimer_R);
 }
 
 
 
 // update tooltip showing value and adjust position while changing fader value
 void Fader::updateTextFloat()
 {
-	if( ConfigManager::inst()->value( "app", "displaydbfs" ).toInt() && m_conversionFactor == 100.0 )
+	if(ConfigManager::inst()->value("app", "displaydbfs").toInt() && m_conversionFactor == 100.0)
 	{
-		s_textFloat->setText( QString("Volume: %1 dBFS").
-				arg( ampToDbfs( model()->value() ), 3, 'f', 2 ) );
+		s_textFloat->setText(QString("Volume: %1 dBFS").
+				arg(ampToDbfs(model()->value()), 3, 'f', 2));
 	}
 	else
 	{
-		s_textFloat->setText( m_description + " " + QString("%1 ").arg( model()->value() * m_conversionFactor ) + " " + m_unit );
```

Comment:
Is it correct to remove the space before `m_unit` here?

Reply:
The space is part of the unit string, because things like "%" are supposed to go without one. Other widgets like knobs don't add a space here, but faders had double spaces because of this.